plugins/outputs/groundwork/groundwork.go

```diff
@@ -5,11 +5,10 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"strconv"
-
 	"github.com/gwos/tcg/sdk/clients"
 	"github.com/gwos/tcg/sdk/transit"
 	"github.com/hashicorp/go-uuid"
+	"strconv"
```

Comment:
I think internal libraries should be kept together.

Reply:
Fixed ✅